pom.xml

```diff
@@ -9,17 +9,32 @@
 
     <properties>
         <java.version>1.8</java.version>
-        <handlebars-java-version>4.0.3</handlebars-java-version>
+        <handlebars-java-version>4.3.1</handlebars-java-version>
+        <handlebars-markdown-java-version>4.2.1</handlebars-markdown-java-version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <powermock.version>1.5.6</powermock.version>
```

Comment:
We don't seem to be using Powermock, so I think this can be removed.

Reply:
removed